Tests/Data/MissionTest-1.0.ckan

```diff
@@ -0,0 +1,18 @@
+{
+    "spec_version"   : "v1.24",
```

Comment:
We'll have to change this to 1.26 after its release

Reply:
Actually, there is absolutely no reason we can't make it v1.25. We aren't doing odd-numbered "development" releases.